src/main/webapp/app/entities/flair-bi/modal/modal-tabs/body-properties.component.html

```diff
@@ -12,9 +12,10 @@
             <label class="control-label flair-model-pad pull-left">Opacity</label>
             <rzslider rz-slider-model="vm.slider.value" rz-slider-options="vm.slider.options"></rzslider>
             <input type="hidden" class="form-control pull-left" ng-model="vm.visual.bodyProperties.opacity">
+
         </div>
-        
-        <div class="row">
+
+        <div class="row"  style="margin-top: 35px;">
```

Comment:
can we move the css to scss file

Reply:
there was no any specific scss file for that , that why i use inline css